examples/server/public/index.html

```diff
@@ -467,6 +479,24 @@
       transcriptUpdate([]);
     }
 
+    const uploadImage = (e) => {
+      e.preventDefault();
+      document.getElementById("fileInput").click();
+      document.getElementById("fileInput").addEventListener("change", function (event) {
+            const selectedFile = event.target.files[0];
+            if (selectedFile) {
+                const reader = new FileReader();
+                reader.onload = function () {
+                    const image_data = reader.result;
+                    params.value = {...params.value, image_data: [
+                      { data: image_data.replace('data:image/png;base64,', ''), id: 10 }] }
```

Comment:
png
any way to make it image type agnostic? like `data:image/*;` ?

Reply:
It can be 

```javascript
+ Image_data.substring(image_data.indexOf("base64,") + 7)
```